hwdef: GEPRCF745BTHD: rely on custom builds for non-BMP280 baros

peterbarker opened the pull request:
this is overflowing - it has an onboard baro, that will have to do..

@andyp1per @Hwurzburg @YI-BOYANG this board did not compile for ArduPlane when support was merged in https://github.com/ArduPilot/ardupilot/pull/27491

YI-BOYANG commented:
Is it possible to add DPS310 and MS5611 drivers? Because our GPS products integrate these barometers.

peterbarker commented:
> Is it possible to add DPS310 and MS5611 drivers? Because our GPS products integrate these barometers.

Yes!  I've added lines to include those and adjusted the comment.

YI-BOYANG commented:
> > Is it possible to add DPS310 and MS5611 drivers? Because our GPS products integrate these barometers.
> 
> Yes! I've added lines to include those and adjusted the comment.

Thanks!

andyp1per commented:
I have one of these so can test now.
